helper/raftutil/msgtypes.go

```diff
@@ -65,6 +65,7 @@ var msgTypeNames = map[structs.MessageType]string{
 	structs.ACLBindingRulesDeleteRequestType:             "ACLBindingRulesDeleteRequestType",
 	structs.NodePoolUpsertRequestType:                    "NodePoolUpsertRequestType",
 	structs.NodePoolDeleteRequestType:                    "NodePoolDeleteRequestType",
+	structs.VarRenewLockRequestType:                      "VarRenewLockRequestType",
```

Comment:
The check fails if this message type is not added, but this message is not intended to make it to raft.